pymhf/core/utils.py

```diff
@@ -1,6 +1,10 @@
 from ctypes import windll, create_unicode_buffer, byref, c_ulong
 from configparser import ConfigParser
 from typing import Optional
+import pywinctl as pwc
+from pywinctl import Window
```

Comment:
this import isn't used

Reply:
I imported the Window class here to avoid an implicit dependency when initializing a class object.

Reply:
fixed